2021-04-11 07:31:20,438 - gravity_core.wlogger - ERROR - Traceback (most recent call last):
  File "/home/watchman/Gravity-Core/gravity_core/wlistener.py", line 102, in broadcastMsgSend
    conn.send(msg)
ConnectionResetError: [Errno 104] Connection reset by peer

2021-04-11 07:43:43,002 - gravity_core.wlogger - ERROR - Traceback (most recent call last):
  File "/home/watchman/Gravity-Core/gravity_core/wlistener.py", line 102, in broadcastMsgSend
    conn.send(msg)
BrokenPipeError: [Errno 32] Broken pipe

2021-04-11 07:43:45,401 - gravity_core.wlogger - ERROR - Traceback (most recent call last):
  File "/home/watchman/Gravity-Core/gravity_core/wlistener.py", line 102, in broadcastMsgSend
    conn.send(msg)
BrokenPipeError: [Errno 32] Broken pipe

2021-04-11 19:13:02,380 - gravity_core.wlogger - ERROR - Выход не зафиксирован фотоэлементом
2021-04-11 19:28:44,332 - gravity_core.wlogger - ERROR - Выход не зафиксирован фотоэлементом
